cmake/Modules/FindSYCLToolkit.cmake

```diff
@@ -96,7 +96,7 @@ find_file(
 
 find_library(
   SYCL_LIBRARY
-  NAMES sycl
+  NAMES sycl-preview
```

Comment:
Port Windows path as in [here](https://github.com/pytorch/pytorch/blob/main/cmake/Modules/FindSYCLToolkit.cmake#L58-L74) to link against `sycl7.lib` instead of `sycl-preview`.

Reply:
Update.
refer to https://github.com/pytorch/pytorch/pull/130110/files#diff-65bed5904d10dccd55edec2d52c25c4ac49db608da9c650ada99cb409675e168R58-R74

Reply:
ditto

Reply:
updated